### ISSUE TITLE
shorter command names? (ngxensite / ngxdissite)

### ISSUE THREAD

**Issue by Fedcomp** [perusio/nginx_ensite#15]
For example apache a2ensite is much shorter than nginx_ensite. When i tried to change it manually (mv nginx_ensite ngxensite) i realized dissite is just a symlink and it's all hardcoded into one script.
So how about shorter command names? apache commands is really good because they are short, i see no reason to make long nginx_ensite, because this script mimics apache version (and apache commands are not intuitive in names), so it should be short as well, and not neccessary intuitive.

**Comment by milabs** [comment]
@Fedcomp Nice catch about the names. It would be nice to see something like `nginx+` and `nginx-`

**Comment by Fedcomp** [comment]
for my use in production, i already changed the name to ngensite/ngdissite, and it seems works without issues.

**Comment by milabs** [comment]
@Fedcomp It's because of action matching string:

~~~
ACTION=$(echo $SCRIPTNAME | awk '$0 ~ /dissite/ {print "DISABLE"} $0 ~ /ensite/ {print "ENABLE"} $0 !~ /(dis|en)site/ {print "UNKNOWN"}')
~~~

https://github.com/perusio/nginx_ensite/blob/master/bin/nginx_ensite#L75

So, any command name that contains `ensite` and `dissite` is valid. Even `a2ensite` :-)

**Comment by Fedcomp** [comment]
yeah i saw that. I would like however if it would be changed officially, or at least add additional symlinks.

**Comment by milabs** [comment]
@Fedcomp Sure.
